src/Containers/NoteForm/NoteForm.js

```diff
@@ -1,47 +1,136 @@
 import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { setError } from '../../actions';
+import API from '../../utils/api';
 
 class NoteForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      ...this.props.note
+      showPopUp: true,
+      id: this.props.id,
+      title: this.props.title,
+      issues: this.props.issues
```

Comment:
We could destructure props and then would have
`this.state = {
id, 
title, 
issues
}`

Reply:
Not on one line like that 😅but GitHub formatted it that way

Reply:
We had it set up with the destructuring initially but I think the problem was that state didn't like that we were mutating props for the issues.